core/src/main/scala/tofu/common/Display.scala

```diff
@@ -0,0 +1,19 @@
+package tofu.common
+import cats.{Show, Eval}
+
+/** Configurable and performant conversion to String */
+trait Display[A] {
+  def displayBuild(precedence: Int, cfg: DisplayConfig, a: A): Eval[Vector[String]]
```

Comment:
Should be Eval[Chain

Reply:
I strongly disagree. 
Using standard collections here is preferred for future support, and Chain most likely has no benefits here

Reply:
I'll stick for Vector for now. We can change it in the future though if it will be necessary due to performance issues etc.